velero.yaml

```diff
@@ -0,0 +1,88 @@
+package:
+  name: velero
+  version: 1.11.1
+  epoch: 0
+  description: Backup and migrate Kubernetes applications and their persistent volumes
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    runtime:
+      - wolfi-base
+      - restic
+
+environment:
+  contents:
+    packages:
+      - busybox
+      - ca-certificates-bundle
+      - build-base
+      - go
+      - git
+
+pipeline:
+  - uses: git-checkout
+    with:
+      tag: v${{package.version}}
+      expected-commit: bdbe7eb242b0f64d5b04a7fea86d1edbb3a3587c
+      repository: https://github.com/vmware-tanzu/velero
+      destination: velero
+
+  - runs: |
+      cd velero
+      mkdir -p ${{targets.destdir}}/usr/bin/
+
+      CGO_ENABLED=0
+      GO111MODULE=on
+      GOOS=$(go env GOOS)
+      GOARCH=$(go env GOARCH)
+      export GOBIN="${{targets.destdir}}"/usr/bin build
+
+      LDFLAGS="-s -w"
+      LDFLAGS="$LDFLAGS -extldflags"
+      LDFLAGS="$LDFLAGS -static"
+      LDFLAGS="$LDFLAGS -X main.version=v${{package.version}}"
+      LDFLAGS="$LDFLAGS -X github.com/vmware-tanzu/velero/pkg/buildinfo.Version=v${{package.version}}"
+
+      # Go flags
+      # -x Print commands
+      # -v Print names of packages
+      # -o output file location
+      # -trimpath Remove all system file paths
+      # -ldflags Args passed to the go link tool
+
+      go build \
+        -x \
+        -v \
+        -o "$GOBIN/velero" \
+        -trimpath \
+        -ldflags "$LDFLAGS" \
+        ./cmd/velero
+
+      go build \
+        -x \
+        -v \
+        -o "$GOBIN/velero-restore-helper" \
+        -trimpath \
+        -ldflags "$LDFLAGS" \
+        ./cmd/velero-restore-helper
+
+  - uses: strip
+
+subpackages:
+  - name: velero-restore-helper
+    description: Velero Restore Helper
+    pipeline:
+      - runs: |
+          set -x
+          mkdir -p "${{targets.subpkgdir}}/usr/bin/"
```

Comment:
nit, but here's a neat little trick we've been using more and more of to avoid needing the `mkdir -p` step: https://github.com/wolfi-dev/os/blob/main/libaec.yaml#L59